kratos/sources/variables.cpp

```diff
@@ -794,6 +795,7 @@ void KratosApplication::RegisterVariables() {
     KRATOS_REGISTER_VARIABLE(PENALTY)
     KRATOS_REGISTER_VARIABLE(NORMAL_STRESS)
     KRATOS_REGISTER_VARIABLE(TANGENTIAL_STRESS)
+    KRATOS_REGISTER_VARIABLE(FLUID_STRESS)
```

Comment:
Do you have to define this in the core? You are not using it there
It seems like an application-specific variable, and the core is already heavy enough

=> if you need it in several apps then just define it in both apps, this works for a while already

Reply:
Put it in the cfd_variables.h.

Reply:
Because this variable might be interesting for many apps., not only the pure fluid ones but even in the fluid particle ones or similar. In my opinion `FLUID_STRESS` is general enough.

Reply:
If you need it in other apps then you can also define it there -> this is possible now with the hash-based variables
This wa we dont habe to overload the core (which is heavy enough already)

E.g. it is not general enough to be used for structures

Reply:
One wont make a difference, but it accumulates ...
Also I think this originated in a time where the only way to share variables was by putting them in the core

either way, this is my opinion, I wont block bcs of this